content/en/docs/reference/glossary/podpreset.md

```diff
@@ -10,9 +10,9 @@ aka:
 tags:
 - operation
 ---
- An API object that injects information such as secrets, volume mounts, and environment variables into pods at creation time.
+ An API object that injects information such as secrets, volume mounts, and environment variables into {{< glossary_tooltip text="Pods" term_id="pod" >}} at creation time.
 
 <!--more--> 
 
-This object chooses the pods to inject information into using standard selectors. This allows the podspec definitions to be nonspecific, decoupling the podspec from environment specific configuration.
+This object chooses the Pods to inject information into using standard selectors. This allows the podspec definitions to be nonspecific, decoupling the podspec from environment specific configuration.
```

Comment:
nit:
Use the field name, `PodSpec` or reword the second sentence to something like:
This allows the Pod specification to be nonspecific and decouples the `PodSpec` definition from the system environment.